python/lsst/pipe/tasks/colorterms.py

```diff
@@ -166,7 +166,7 @@ def getColorterm(self, filterName, refCatName, doRaise=True):
             ctDictConfig = self.data.get(refCatName)
             if ctDictConfig is None:
                 # try glob expression
-                matchList = [glob for glob in self.data if fnmatch.fnmatch(refCatName, glob)]
+                matchList = [glob for glob in self.data if fnmatch.fnmatch(glob, refCatName)]
```

Comment:
I'm not sure this was wrong.  According to documentation here:
https://docs.python.org/2/library/fnmatch.html
the glob should be the second argument.  Maybe I don't understand what is in glob.

```
>>> import fnmatch
>>> print fnmatch.fnmatch('sdss-dr8', 'sdss-*')
True
>>> print fnmatch.fnmatch('sdss-*', 'sdss-dr8')
False
```